### PR TITLE
Added p:archive-manifest

### PR DESCRIPTION
**Pull request by eriksiegel**
See #115

### CODE REVIEW

**Comment by eriksiegel** [comment]
If we do this, the same mechanism should apply to p:archive and p:unarchive...???

Honestly, I don't think its worth the burden on the implementers. Sounds like a very fringe case to me. Any opinions?

**Comment by xml-project** [comment]
The phrase "implementations should try to infer the archive format from the source document’s content-type property" is vague enough in my opinion to hint on a quality of implementation. I would prefer to have this phrase, because if it is missing, the implementation might be *required* to use the default value, even if it knows what to do with "application/zip".

**Comment by eriksiegel** [comment]
> the implementation might be required to use the default value, even if it knows what to do with "application/zip"

But the default *is* `application/zip` (zip is the default value for the format option). I'm not sure what you mean

**Comment by eriksiegel** [comment]
I tried to process the comments made at our last editorial meeting in the p:archive-manifest description. Please have a look.

If this is ok, I'll revisit and rework the p:archive/unarchive likewise.

File: steps/src/main/xml/steps/archive-manifest.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        supported.</impl> <error code="C0081">It is a <glossterm>dynamic error</glossterm> if the format of the
+          archive does not match the format as specified in the <option>format</option>
+          option.</error></para>
+    </listitem>
+    <listitem>
+      <para>If no <option>format</option> option is specified or its value is the empty sequence,
+        the archive's format will be determined by the step, using the <code>content-type</code>
+        document-property of the document on the <port>source</port> port and/or by inspecting its
+        contents. <impl>It is <glossterm>implementation-defined</glossterm> how the step determines
+          the archive's format.</impl> Implementations <rfc2119>should</rfc2119> recognize archive's
```
Comment (gimsieke) [inline steps/src/main/xml/steps/archive-manifest.xml]: recognize archive's → recognize archives

File: steps/src/main/xml/steps/archive-manifest.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      <para>If the <option>format</option> option is specified, this determines the format of the
+        archive. Implementations <rfc2119>must</rfc2119> support the <biblioref linkend="zip"/>
+        format, specified with the value <code>zip</code>. <impl>It is
+            <glossterm>implementation-defined</glossterm> what other formats are
+        supported.</impl> <error code="C0081">It is a <glossterm>dynamic error</glossterm> if the format of the
+          archive does not match the format as specified in the <option>format</option>
+          option.</error></para>
+    </listitem>
+    <listitem>
+      <para>If no <option>format</option> option is specified or its value is the empty sequence,
```
Comment (gimsieke) [inline steps/src/main/xml/steps/archive-manifest.xml]: or its value → or if its value
(I’m not too confident about this change, but it sounds better to me.)

File: steps/src/main/xml/steps/archive-manifest.xml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    archive cannot be understood, determined and/or processed.</error></para>
+
+  <para>The <option>parameters</option> option can be used to supply parameters to control the
+    archive manifest generation. <impl>The semantics of the keys and the allowed values for these
+      keys are <glossterm>implementation-defined</glossterm>.</impl>
+    <error code="C0079">It is a <glossterm>dynamic error</glossterm> if the map
+        <option>parameters</option> contains an entry whose key is defined by the implementation and
+      whose value is not valid for that key.</error></para>
+
+  <para>The generated archive has the format as described in <xref linkend="cv.archive-manifest"/>.
```
Comment (gimsieke) [inline steps/src/main/xml/steps/archive-manifest.xml]: generated archive → result document

Comment (eriksiegel) [reply steps/src/main/xml/steps/archive-manifest.xml]: ok, done

Comment (gimsieke) [reply steps/src/main/xml/steps/archive-manifest.xml]: The sentence still seems to read “The generated archive has the format as described in …”. But there is no generated archive. You can call it “generated archive manifest” or, as I proposed, “result document”.

**Comment by eriksiegel** [comment]
Sorry, forgot this, will fix.

**Pull request merged**